src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -291,9 +288,9 @@ public enum RolesMappingResolution {
     public static final Boolean SECURITY_SYSTEM_INDICES_ENABLED_DEFAULT = false;
     public static final String SECURITY_SYSTEM_INDICES_KEY = "plugins.security.system_indices.indices";
     public static final List<String> SECURITY_SYSTEM_INDICES_DEFAULT = Collections.emptyList();
-
-    public static final String TENANCY_PRIVATE_TENANT_NAME = "private";
-    public static final String TENANCY_GLOBAL_TENANT_NAME = "global";
+    public static final String SYSTEM_INDEX_PERMISSION = "system:admin/system_index";
+    public static final String TENANCY_PRIVATE_TENANT_NAME = "Private";
```

Comment:
Is this supposed to be swapped to capital?

Reply:
Not sure... It came from another commit/user,  I might have been changed again on Main after I synched. I will revert it

![image](https://github.com/opensearch-project/security/assets/128482925/d61c64fe-299a-47fa-9a39-6bf0385e570f)